docs-ref-services/storage.md

```diff
@@ -25,39 +25,71 @@ Use the Azure Storage client module to:
 
 Create, update, and manage Azure Storage accounts and query and regenerate access keys from your Node.js apps with the management modules.
 
-## Install modules with npm
+## Client Package
 
-Use npm to install the Azure storage client or management modules.
-
-### Client
+### Install packages with npm
 
 ```bash
 npm install azure-storage
 ```
 
-### Management
+### Example
+
+Create a storage container and upload a local file *data.txt* to it.
+
+```javascript
+const azure = require('azure-storage');
+const blobService = azure.createBlobService(storageConnectionString);
+
+const container = 'taskcontainer';
+const task = 'taskblob';
+const filename = 'data.txt';
+
+blobService.createContainerIfNotExists(container, error => {
+  if (error) return console.log(error);
+  blobService.createBlockBlobFromLocalFile(
+    container,
```

Comment:
did "prettier" make these parameters go to separate lines? seems like they should fit ok on one